hackathon_site/hardware/views.py

```diff
@@ -101,7 +109,10 @@ def get_permissions(self):
         if self.request.method == "POST":
             return [UserHasProfile()]
         if self.request.method == "GET":
-            return [FullDjangoModelPermissions()]
+            if self.request.user.has_perm("hardware.change_order"):
+                return [FullDjangoModelPermissions()]
+            else:
+                return [UserHasProfile()]
         return [permissions.IsAuthenticated()]
```

Comment:
Why would you think the return is useless? Do you mean also remove the "else" statement together?

Reply:
I originally thought it was useless because we only have post and get requests for order list view but on second thought I'll keep it in in case we make other requests